ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -140,7 +140,12 @@ done
 # Snapshot iptables-save on each node for debugging possible kube-proxy issues
 oc --insecure-skip-tls-verify get --request-timeout=20s -n openshift-sdn -l app=sdn pods --template '{{ range .items }}{{ .metadata.name }}{{ "\n" }}{{ end }}' > /tmp/sdn-pods
 while IFS= read -r i; do
-  queue ${ARTIFACT_DIR}/network/iptables-save-$i oc --insecure-skip-tls-verify rsh --timeout=20 -n openshift-sdn -c sdn $i iptables-save -c
+  queue ${ARTIFACT_DIR}/network/iptables-save-$i oc --insecure-skip-tls-verify rsh --request-timeout=20s -n openshift-sdn -c sdn $i iptables-save -c
+done < /tmp/sdn-pods
+
+# Snapshot all used ports on each node.
+while IFS= read -r i; do
+  queue ${ARTIFACT_DIR}/network/ss-$i oc --insecure-skip-tls-verify rsh --request-timeout=20s -n openshift-sdn -c sdn $i ss -apn
```

Comment:
Is this something we should be doing for `openshift-ovn-kubernetes` as well? I know we haven't exactly been gathering the iptables for ovn-k though that would be great... but its probably not exactly related to this PR unless the bug https://bugzilla.redhat.com/show_bug.cgi?id=1990662 happens irrespective of the networking plugin?

Reply:
I think my BZ is not related to any particular network plugin - how would a network plugin affect which ports on nodes are used?
I reworked the code a bit to collect network info from both OVN and SDN pods, one of them should succeed.